src/components/UI/LiveHelpButton/LiveHelpButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import LiveHelpIcon from '@material-ui/icons/LiveHelp';
+
+import IconButton from '../IconButton';
+import messages from '../../Settings/Settings.messages';
+
+const propTypes = {
+  /**
+   * The component used for the root node. Either a string to use a DOM element or a component.
+   */
+  component: PropTypes.object,
+  /**
+   * If true, back button is disabled
+   */
+  disabled: PropTypes.bool,
+  /**
+   * @ignore
+   */
+  intl: intlShape.isRequired,
+  /**
+   * Callback fired when LiveHelp button is clicked
+   */
+  onClick: PropTypes.func
+};
+
+function LiveHelpButton(props) {
+  const { intl, ...other } = props;
+  const label = intl.formatMessage(messages.userLiveHelp);
+
+  return (
+    <IconButton id="liveHelp-button" label={label} {...other}>
```

Comment:
I think we can delete the `id`? Didn't find anyone using it (not CSS or JS)

Reply:
@tomivm what do you think about that?

Reply:
@shayc is correct! what i mean we no need that id.

Reply:
Ok! I deleted it!